src/lib/formValidator.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  IFieldValidator,
+  IFieldValidationResponse,
+  fieldValidatorLookup,
+} from "./fieldValidators";
+
+export interface IFormData {
+  [key: string]: string;
+}
+
+export interface IFieldValidatorLookup {
```

Comment:
Again does this need to be an interface.  It could be `Record<string, IFieldValidator>` or if you wanted the type I would be tempted to define it as:

```typescript
export type IFieldValidatorLookup = Record<string, IFieldValidator>;
```